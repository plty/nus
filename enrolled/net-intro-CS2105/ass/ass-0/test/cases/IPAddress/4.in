11111111110101010000010100010001
