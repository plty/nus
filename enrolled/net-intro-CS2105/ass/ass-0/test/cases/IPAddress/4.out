255.213.5.17
